packages/dev/docs/pages/releases/2021-02-16.mdx

```diff
@@ -0,0 +1,144 @@
+- Retain `ComboBox` selected key `onBlur` if value matches - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/1541)
+- Prevent left/right arrow keys from scrolling containers in `Picker` - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/1540)
+- Add check for ref in `useSelectableList` - [@AndyOGo](https://github.com/AndyOGo) - [PR](https://github.com/adobe/react-spectrum/pull/1550)
+
+
+## Docs
+- Fix typo in contribution guide - [@stefanprobst](https://github.com/stefanprobst) - [PR](https://github.com/adobe/react-spectrum/pull/1319)
+- Add the default for `level` prop in `Heading`  - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/1349)
+- Add example of passing a ref to `useButton` - [@toshi1127](https://github.com/toshi1127) - [PR](https://github.com/adobe/react-spectrum/pull/1298)
+- Improve `useCheckboxGroupItem` examples - [@stefanprobst](https://github.com/stefanprobst) - [PR](https://github.com/adobe/react-spectrum/pull/1336)
+- Update links for `TextArea` guidelines - [@matthewdeutsch](https://github.com/matthewdeutsch) - [PR](https://github.com/adobe/react-spectrum/pull/1357)
+- Fix typo in `IllustratedMessage` example - [@travigd](https://github.com/travigd) - [PR](https://github.com/adobe/react-spectrum/pull/1410)
+- Update example in `useRadio` to include a ref - [@matamatanot](https://github.com/matamatanot) - [PR](https://github.com/adobe/react-spectrum/pull/1370)
+- Add anatomy diagrams to aria hooks - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/1452)
+- Update prop description in `useMenuSection` - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/1461)
+- Fix error in storybook - [@tomsontom](https://github.com/tomsontom) - [PR](https://github.com/adobe/react-spectrum/pull/1491)
+- Add `useComboBox` hook documentation to react-aria - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/1276)
+
+
+## Under construction
+
+Pre-release versions of the following components have been released. Please feel free to try them out, and report any issues you encounter.
+
+- NumberField - [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/NumberField.html), [React Aria](https://react-spectrum.adobe.com/react-aria/useNumberField.html)
+- ColorWheel - [React Aria](https://react-spectrum.adobe.com/react-aria/useColorWheel.html)
+- ColorSlider - [React Aria](https://react-spectrum.adobe.com/react-aria/useColorSlider.html)
+- ColorField - [React Aria](https://react-spectrum.adobe.com/react-aria/useColorField.html)
+- ComboBox - [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/ComboBox.html), [React Aria](https://react-spectrum.adobe.com/react-aria/useComboBox.html)
```

Comment:
ComboBox had already been released as an alpha in a previous release, do we want to include it here?

Reply:
yeah we made updates and no harm in calling out again for people to try

Reply:
sounds good